ssv/spectest/tests/runner/pre_consensus_justifications/decided_instance.go

```diff
@@ -0,0 +1,104 @@
+package pre_consensus_justifications
+
+import (
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/ssv/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// DecidedInstance tests a valid pre-consensus justification for a decided instance
+func DecidedInstance() tests.SpecTest {
+	ks := testingutils.Testing4SharesSet()
+
+	msgF := func(obj *types.ConsensusData, id []byte) *qbft.SignedMessage {
+		fullData, _ := obj.Encode()
+		root, _ := qbft.HashDataRoot(fullData)
+		msg := &qbft.Message{
+			MsgType:    qbft.ProposalMsgType,
+			Height:     qbft.FirstHeight,
+			Round:      qbft.FirstRound,
+			Identifier: id,
+			Root:       root,
+		}
+		signed := testingutils.SignQBFTMsg(ks.Shares[1], 1, msg)
+		signed.FullData = fullData
+
+		return signed
+	}
+
+	return &tests.MultiMsgProcessingSpecTest{
+		Name: "pre consensus decided instance",
+		Tests: []*tests.MsgProcessingSpecTest{
+			{
+				Name:   "sync committee aggregator selection proof",
+				Runner: decideFirstHeight(testingutils.SyncCommitteeContributionRunner(ks)),
+				Duty:   &testingutils.TestingSyncCommitteeContributionDuty,
+				Messages: []*types.SSVMessage{
+					testingutils.SSVMsgSyncCommitteeContribution(msgF(testingutils.TestContributionProofWithJustificationsConsensusData(ks), testingutils.SyncCommitteeContributionMsgID), nil),
```

Comment:
essentially this is a qbft propose message. In the debugger I saw that it invokes the creation and broadcast of a non-nil prepare message.
But OutputMessages don't show it. So I am confused...